### ISSUE TITLE
custom-domain create fail.

### ISSUE THREAD

**Issue by KevinZonda** [Azure/azure-cli#10529]
## Describe the bug

**Command Name**
`az cdn custom-domain create`

**Errors:**
```
Deployment failed. Correlation ID: 9a64caee-b633-4cd0-a858-af2d44725c84. Sorry, it looks like there was an error on our end. Please contact Support if you keep having this problem.
```

## Produce:

- Deploy cdn and endpoint in the portal.
- `az cdn custom-domain create --endpoint-name {} --hostname {} --profile-name {} -g {} -n {}`

## Expected Behavior

Deployment seems be successful(in portal). But failed in Azure-cli return.

## Environment Summary
```
Windows-10-10.0.17763-SP0
Python 3.6.6
Shell: powershell.exe

azure-cli 2.0.72 *

Extensions:
front-door 1.0.0

```

**Comment by lsmith130** [comment]
I'm working on a fix for this in conjunction with #9894, which should be complete next week.

**Comment by lsmith130** [comment]
Fixed in #12648.
